src/sidebar/components/MarkdownView.js

```diff
@@ -1,16 +1,16 @@
-import classnames from 'classnames';
 import { useEffect, useMemo, useRef } from 'preact/hooks';
 
 import { replaceLinksWithEmbeds } from '../media-embedder';
 import { renderMathAndMarkdown } from '../render-markdown';
 
+import StyledText from './StyledText';
+
 /**
  * @typedef MarkdownViewProps
  * @prop {string} markdown - The string of markdown to display
- * @prop {Record<string,string>} [textStyle] -
- *   Additional CSS properties to apply to the rendered markdown
- * @prop {Record<string,boolean>} [textClass] -
- *   Map of classes to apply to the container of the rendered markdown
+ * @prop {string} [classes]
+ * @prop {Record<string,string>} [style]
```

Comment:
I renamed and simplified these props to be more consistent with how we handle classNames and style on other presentational components.